tools/server.pem

```diff
@@ -0,0 +1,49 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
We should insert a comment telling this is a test file (and has been put here on purpose).